WPMedia/engine-theme-sdk
# PEN-1736 SVG Icon updates to provide context prop

## Description

Update icon components to provide a context prop, to allow for the consumer to decide what context the icon will be used in. Thus allowing the SVG to have the correct attributes for accessibility.

## Jira Ticket
- [PEN-1736](https://arcpublishing.atlassian.net/browse/PEN-1736)

## Test Steps

- Verify icon components still render as before
- Verify icon components default to presentational and include the correct attributes
- Verify if an icon is used in the `image` context the correct attributes are applied


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).
